jfrog-cli/docs/artifactory/gradle/help.go

```diff
@@ -9,3 +9,12 @@ const Arguments string = `	tasks and options
 
 	config file path
 		Path to a configuration file generated by the "jfrog rt gradlec" command.`
+
+const EnvVar string = `	JFROG_CLI_JCENTER_REMOTE_SERVER
+		Configured Artifactory server ID from which to download the jar needed by the gradle command.
```

Comment:
I'm nor sure. This may be confusing because of the following reasons:

This env var should include a configured server ID, and *bintray* is not a server ID. If by any chance someone adds a server named *bintray, then this env var's default value will be incorrect.